docs/authentication/enterprise-connections/oidc/custom-provider.mdx

```diff
@@ -29,33 +29,48 @@ To make the setup process easier, it's recommended to keep two browser tabs open
 
   1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
   1. Select **Add connection** and select **For specific domains**.
-  1. Under **Third party**, select **OIDC**.
+  1. Under **Third party**, select **OpenID Connect (OIDC)**.
   1. Add the **Name** of the connection.
   1. Add the **Key** of the provider. This is the provider's unique identifier (cannot be changed after creation).
   1. Add the **Specific Domain** that you want to allow this connection for. This is the domain of the users you want to allow to sign in to your app.
   1. Select **Add connection**. You will be redirected to the connection's configuration page.
 
-  The next steps involve configuring both your IdP and Clerk. Keep both dashboards open as you'll need to reference them.
-
   ### Configure your IdP
 
   1. If necessary, create a new app in your IdP.
-  1. In the connection's configuration page in the Clerk Dashboard, copy the **Authorized redirect URI**.
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
```

Comment:
they should already be there according to the last step in the previous step. so the copy before was correct